### ISSUE TITLE
Evaluate green skill measures

### ISSUE THREAD

**Issue by lizgzil** [nestauk/dap_prinz_green_jobs#72]
For a sample of skills find:
- is the skill extracted well?
- if green, is it mapped to green ESCO well?
- is it mapped to the full ESCO well?

Started a labelling document on our mixed sample here https://docs.google.com/spreadsheets/d/18Qdq5WIplVOeRtpswjderAEVXBEZ5L4VXQa0tulRx0k/edit#gid=1139196409

More keen on labelling green skills than non-green, but I think its nice to have a bit of a mix? Also there is some overlap with the evaluation of the skills extraction algorithm evaluation, but it might be good to have a new evaluation of green skills specifically as they may perform differently.

Note, we have the test metrics for the green skill / not green skill algorithm [here](https://s3.console.aws.amazon.com/s3/object/prinz-green-jobs?region=eu-west-2&prefix=outputs/models/green_skill_classifier/green_skill_classifier_20230906_results.json)

In short: 
```
"green": {"precision": 0.904, "recall": 0.863, "f1-score": 0.883, "support": 175},
"not_green": {"precision": 0.908, "recall": 0.937, "f1-score": 0.922, "support": 254}
```

**Comment by india-kerle** [comment]
merged branch - closing issue
